zstacklib/zstacklib/utils/pci.py

```
@@ -224,6 +224,33 @@ def get_pci_passthrough_mapping(vm_dom):
     return pci_mapping
 
 
+def get_mdev_passthrough_mapping(vm_dom):
+    mdev_mapping = {}
+    xml_tree = ET.fromstring(vm_dom.XMLDesc())
```

Ruff (0.14.11)

Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)